### PR TITLE
Update ind. call/jmp identification and its placement

### PR DESCRIPTION
**Pull request by ved-rivos**
Remove duplication of text by referring to the pseudocode.

### CODE REVIEW

**Comment by ved-rivos** [comment]
@kacouane - I updated the pseudo-code in your pull request and its placement. Please review.

File: cfi_intro.adoc [hunk 1]
```diff
@@ -51,39 +51,19 @@ not the conventional link register is a indirect jump.
...
 conventions and hints. 
 
-The term call is used to refer to a `JAL` or `JALR` instruction (and their
-compressed forms `C.JAL` and `C.JALR`) with `x1` or `x5` as the `rd`. A call
-using `JAL` and `C.JAL` is termed a direct call. A call using `JALR` and
-`C.JALR` is termed an indirect call.
+The term call is used to refer to a `JAL` or `JALR` instruction with a link
+register as destination i.e., `rd != x0`. Conventionally the link register is
+`x1` or `x5`. A call using `JAL` or `C.JAL` is termed a direct call. A call
+using `JALR` or `C.JALR` is termed an indirect call.
```
Comment (kwalker27) [inline cfi_intro.adoc:57]: Maybe include the expansion of `c.jalr` to `jalr x1, rs1` (like you do for c.jr below) to make it clear why it's always considered a call?

File: cfi_intro.adoc [hunk 2]
```diff
@@ -128,6 +108,27 @@ for indirect calls and jumps in a program. Compiler is expected to emit a `lpcll
...
+[[IND_CALL_JMP]]
+.Indirect call/jump determination
+[source, text]
+----
+    if ( ((JALR && rd == x0) || (C.JR)) && (rs1 != x1) && (rs1 != x5) )
+        is_indirect_jump = 1;
+    else
+        is_indirect_jump = 0;
+
+    if ( (JALR && rd != x0) || (C.JALR) )
```
Comment (kwalker27) [inline cfi_intro.adoc:123]: It's really `rd != x0` rather than checking for `x1` or `x5`?

Comment (ved-rivos) [reply cfi_intro.adoc:123]: Yes,

File: cfi_intro.adoc (hunk 1 above)
Comment (ved-rivos) [reply cfi_intro.adoc:57]: Okay.

**Pull request merged**